app/views/submissions/_release_grades.html.erb

```diff
@@ -0,0 +1,13 @@
+<span id="release-grades">
+  <% unless released(@assessment, @submission) %>
+    <%= link_to(releaseSubmission_course_assessment_submission_path(@course, @assessment, @submission),
+                class: "btn small", title: "Release Grades") do %>
+    Release Grades
```

Comment:
I feel like calling the button "Release Grades" could be confusing for the instructor b/c it sounds like they are releasing the grades for all students. Possible alternative could be "Release grade" but idk if that captures what we want either

Reply:
I think it would be helpful to add a title (which shows when you hover over the button) to specify that it is only releasing the grade for the current student